app/scripts/views/components/scatterPlot/scatterPlot.js

```diff
@@ -9,6 +9,17 @@
     var chartId_;
     var data_;
     var opts_;
+    var getQtipString = function(_data) {
+      var toReturn = 'Cancer Study:' + _data.study_id + '<br>Sample Id: ' +
+        _data.sample_id + '<br>CNA fraction: ';
+      if (isNaN(_data.cna_fraction)) {
```

Comment:
The situation is if the data isNaN, the case will not even be shown in scatter plot. It's ok to have this condition, but I don't  it will be used.